src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationOutputMappingPresenter.cs

```diff
@@ -96,15 +96,12 @@ public void Refresh()
          this.DoWithinLatch(() =>
          {
             UpdateCache();
-            updateOutputMapppingList();
-            _view.BindTo(_allOutputMappingDTOs);
+            updateOutputMappingList();
          });
+         _view.BindTo(_allOutputMappingDTOs);
```

Comment:
That's interesting here. Why outside?

Reply:
ah, that's because within the BindTo the selection of mapping can change. I needed to BindTo outside the latch because the latching prevents the Select event to fire.